classifiers/image/CLIPZeroShotClassifier/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM jinaai/jina AS base
```

installation of pytorch is slow while jina installation is fast. Let's use pytorch image as the base.
```suggestion
FROM pytorch/pytorch:1.7.1-cuda11.0-cudnn8-runtime

```